internal/cli/option/flag.go

```diff
@@ -98,11 +142,25 @@ func OldAlias(fs *pflag.FlagSet, stage *string, usage string) {
 	fs.StringVar(stage, OldAliasFlag, "", usage)
 }
 
+func Password(fs *pflag.FlagSet, password *string, usage string) {
```

Comment:
```suggestion
// Password adds the PasswordFlag to the provided flag set.
func Password(fs *pflag.FlagSet, password *string, usage string) {
```

Reply:
Fixed in a051595b97541cab94bb7e8f3be72f480a7f8022